draft-ietf-rats-corim.md

```diff
@@ -996,6 +1000,36 @@ measurements for the Target Environment.
 {::include cddl/coswid-triple-record.cddl}
 ~~~
 
+#### Conditional Reference Series Triple {#sec-comid-triple-cond-ref}
+
+A Conditional Reference Series triple associates conditional Reference Value measurements,
+and optionally Endorsement measurements, to a Target Environment, where the endorsed measurements
+are accepted if all of the reference measurements are accepted.
+
+The triple subject is a compound statement consisting of a Target Environment defined
+by `environment-map`, and initial Reference Values defined by `measurement-map`,
+that MUST be matched to Evidence before evaluating the triple object.
+If the initial Reference Values do not match Evidence, the triple is not matched
+and no Claims are accepted.
+
+The `authorized-by` value in `measurement-map`, if present, applies to all measurements
+in the triple including those in `conditional-series-record` records.
+
+The triple object is a series of additional measurements expressed as an array of
```

Comment:
In my view we should have a simplified triple object that simply says that accept these values when the `subject matches`.

This PR makes the individual conditional checking within the object far too  complicated to have it in the main specifiction